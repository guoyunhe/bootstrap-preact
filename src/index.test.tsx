import { render, screen } from '@testing-library/preact';
import { PreactBootstrap } from '.';

describe('PreactBootstrap', () => {
  it('render', async () => {
    render(<PreactBootstrap>foobar</PreactBootstrap>);
    await screen.findByText('foobar');
  });
});
