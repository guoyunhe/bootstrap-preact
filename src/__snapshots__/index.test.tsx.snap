// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`members > render 1`] = `
{
  "Button": [Function],
  "Collapse": [Function],
}
`;
